benchmarks/benchmarks.py

```diff
@@ -0,0 +1,86 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+import numpy as np
+import scipy.stats as st
+
+
+class Hist:
+    def time_histogram(self):
+        try:
+            data = np.random.rand(10000, 1000)
```

Comment:
Small thing. In these checks the data line is repeated but it seems it doesn't have to be

Reply:
It can be changed. It's just that asv takes into account the data initialization time as well so I had to include it in both blocks so as to nullify this effect.

Reply:
Got it. Today I learned something